pkg/resources/row_access_policy_acceptance_test.go

```diff
@@ -23,13 +24,29 @@ func TestAcc_RowAccessPolicy(t *testing.T) {
 	id := acc.TestClient().Ids.RandomSchemaObjectIdentifier()
 	resourceName := "snowflake_row_access_policy.test"
 
-	m := func() map[string]config.Variable {
-		return map[string]config.Variable{
-			"name":     config.StringVariable(id.Name()),
-			"database": config.StringVariable(acc.TestDatabaseName),
-			"schema":   config.StringVariable(acc.TestSchemaName),
-		}
+	body := "case when current_role() in ('ANALYST') then true else false end"
+	changedBody := "case when current_role() in ('CHANGED') then true else false end"
+	argument := []sdk.RowAccessPolicyArgument{
+		{
+			Name: "N",
+			Type: string(sdk.DataTypeVARCHAR),
+		},
+		{
+			Name: "V",
+			Type: string(sdk.DataTypeVARCHAR),
+		},
+	}
+	changedArgument := []sdk.RowAccessPolicyArgument{
```

Comment:
how is it changed comparing to the argument above?

Reply:
Done.